qwandaq/src/main/java/life/genny/qwandaq/managers/capabilities/CapabilitiesManager.java

```diff
@@ -237,18 +234,36 @@ public BaseEntity addCapabilityToBaseEntity(String productCode, BaseEntity targe
 		return targetBe;
 	}
 
+	public BaseEntity removeCapabilityFromBaseEntity(String productCode, BaseEntity targetBe, String capabilityCode) {
+		capabilityCode = cleanCapabilityCode(capabilityCode);
+		Attribute attr = qwandaUtils.getAttribute(productCode, capabilityCode);
+		return removeCapabilityFromBaseEntity(productCode, targetBe, attr);
+	}
+
+	public BaseEntity removeCapabilityFromBaseEntity(String productCode, BaseEntity targetBe, Attribute capabilityAttribute) {
+		if (capabilityAttribute == null) {
+			throw new ItemNotFoundException(productCode, "Capability Attribute");
+		}
+
+
+		targetBe.addAttribute(capabilityAttribute, 0.0, "[]");
```

Comment:
This is legacy code and presented a bug in lojing-prod. The fix is in 10.3.0-lojing as of 23/01 @ 4:30p